app/controllers/reviews_controller.rb

```diff
@@ -11,6 +11,11 @@ def new
     @review = Review.new
   end
 
+  def edit
+    @neighborhood = Neighborhood.find(params[:neighborhood_id])
+    @review = Review.find(params[:id])
```

Comment:
you could also use @neighborhood.reviews.find(params[:id]) - this would constrain the possible records to only those that are associated with the neighborhood